docs/resources/configuration_variable.md

```diff
@@ -34,10 +35,11 @@ resource "env0_configuration_variable" "drop_down" {
 ### Required
 
 - **name** (String) name to give the configuration variable
-- **value** (String) value for the configuration variable
+- **value** (String, Sensitive) value for the configuration variable
 
 ### Optional
 
+- **description** (String) a description of the variables
```

Comment:
Please add that we limit it to 255 chars

Reply:
66d474e675695744c8b07b2d6603e60d53bbae43